src/components/PostPage/PostPage.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import dummyData from '../../dummy-data';
+import PostContainer from '../PostContainer/PostContainer';
+import SearchBar from '../SearchBar/SearchBar';
+// import withAuthenticate from './components/authentication/withAuthenticate';
+
+class PostPage extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      posts: [],
+      filtered: []
+    };
+  }
+
+  addComment = (newComment, index) => {
+    let updatedPost = [...this.state.posts]
+    updatedPost[index].comments = [...updatedPost[index].comments, newComment]
+    this.setState({ posts: updatedPost })
```

Comment:
addComment is a method we pass down in order to update our posts array here in PostPage, you should do the same sort of thing with likes.